automation/WinFormsAutomationApp/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Selenium.Support" version="3.3.0" targetFramework="net452" />
```

Comment:
Please remove/undo adding these Selenium packages now that the browser automation bits have been moved into the TestAutomation repo.